src/CalculatorUITestFramework/CalcHistoryFlyout.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Appium;
+using OpenQA.Selenium.Appium.Windows;
+using System.Collections.ObjectModel;
+using OpenQA.Selenium.Interactions;
+using System.Windows.Input;
+
+namespace CalculatorUITestFramework
+{
+    public class CalcHistoryFlyout
+    {
+        private WindowsDriver<WindowsElement> session => WinAppDriver.Instance.CalculatorSession;
+        public HistoryPanel HistoryPanel = new HistoryPanel();
+        public WindowsElement Header => this.session.TryFindElementByAccessibilityId("Header");
+        public WindowsElement AppName => this.session.TryFindElementByAccessibilityId("AppName");
+        public WindowsElement HistoryFlyout => this.session.TryFindElementByAccessibilityId("HistoryFlyout");
+        public AppiumWebElement ListViewItem => WinAppDriver.Instance.CalculatorSession.FindElementByClassName("Popup").FindElementByClassName("ListViewItem");
```

Comment:
>FindElementByClassName [](start = 88, length = 22)

Finding by classname is something that we should probably try to avoid as if someone adds another popup or listviewitem to the product you might find the wrong one.

Reply:
These elements do not have automation IDs.  Would it be possible to give them automation IDs?